### PR TITLE
Re-wording the role definition in the context of Organization and Env…

### PR DESCRIPTION
**Pull request by laudares**
…ironment-specific levels.

### CODE REVIEW

File: content/general-reference/rbac.md [hunk 1]
```diff
@@ -16,21 +16,22 @@ please make sure you check out the
 
 ## Roles
 
-A *Role* is a set of actions that is granted to a user in a context. For example,
+A *Role* is a set of actions (or "privileges") that is granted to a user in a certain context. For example,
 a role of *Read-Only* in the "Production" environment grants read access to a user for
 all non-sensitive data in the environment.
 
-Roles are assigned in two ways:
+However, you cannot directly assign a role to a particular user: roles are assigned to teams instead. In order to have a user affected by a certain role we must make that user a member of a team for which this role has been assigned to.
```
Comment (seaparks) [inline content/general-reference/rbac.md]: I'd phrase this in a more positive manner. Rather than "However, you cannot directly assign a role..." I would rephrase it something like "Roles are assigned to Teams, and Uses are added to teams," etc.

File: content/general-reference/rbac.md [hunk 2]
```diff
@@ -16,21 +16,22 @@ please make sure you check out the
...
+Roles are assigned at two levels:
 
-By assigning a *role* to a *team*, you are controlling the default privileges on that team.
+- at the Organization level, where we assign a *role* to a *team* in a more general way
+- at the Environment level, where we assign a *role* to a specific *environment-team* pair, thus making it specific to the environment at hand.
+
+By assigning a *role* to a *team* at the Organization level (click on "Settings" and then "Teams" under the "Organization Settings" section) you are controlling the *default* privileges on that team.
 These actions include things like, adding users to your organization, inviting people to a team,
-managing teams or creating new environments.
+managing teams or creating new environments. (See section "Privileges" below for a detailed list).
```
Comment (seaparks) [inline content/general-reference/rbac.md]: While you're in here, I'd fix this wrong use of the comma: it should be "These actions include things like adding users to your organization, inviting people to a team, managing teams, or creating new environments."

File: content/general-reference/rbac.md [hunk 3]
```diff
@@ -16,21 +16,22 @@ please make sure you check out the
...
 
-By assigning a role to an environment-team couple, you can do things like granting
-different privileges per environment to a Team. For example: if a team of "Developers",
+By assigning a role to an environment-team pair (by first selecting the environment you want to affect in the upper left corner of the UI, then clicking in "Settings" and then "Teams" under the "Environment Settings" section), you can override the default role of a team an grant specific privileges on a per environment basis. For example: if a team of "Developers",
```
Comment (seaparks) [inline content/general-reference/rbac.md]: One thing I forgot to mention (my fault): we try to keep lines no longer than 80 - 90 characters per the age-old convention. Can you split up this line and the line above talking about roles?

Environment should be capitalized where it appears, as it's capitalized elsewhere.
"default role of a team an grant" - should be "and grant".

I would take out the parenthetical comment ("by first selecting the environment...") and add that after the sentence. It's long enough that I think it disrupts the flow of describing the environment team pair.

**Comment by laudares** [comment]
@seaparks - I've implemented all your suggestions, except for this one:
> Environment should be capitalized where it appears, as it's capitalized elsewhere.
It only shows with a capital "E" in the 2nd paragraph (when referring to Environments & Teams) and when I highlighted the differences at the  "Organization level" and the "Environment level". Where exactly you think we should use Environment with a capital "E" ?

**Comment by seaparks** [comment]
@laudares Actually, you're right and I misread. We do use lowercase E everywhere. In this case, somewhere we say "...at the Environment level, where we assign..." so I'd switch that to lowercase E. We also seem to only use lowercase T for "Team," so it makes sense to use the lowercase E.

**Comment by laudares** [comment]
@seaparks - I changed all mentions to environment, organization, and team that were not related to a link or a page that displays them with an uppercase.
